pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -78,6 +81,32 @@ stdenv.mkDerivation rec {
       inherit (addOpenGLRunpath) driverLink;
     })
     ./libglvnd-headers.patch
+  ] ++ lib.optionals stdenv.isDarwin [
+    (fetchpatch {
+      url = "https://github.com/WebKit/WebKit/commit/94cdcd289b993ed4d39c17d4b8b90db7c81a9b10.diff";
+      sha256 = "sha256-ywrTEjf3ATqI0Vvs60TeAZ+m58kCibum4DamRWrQfaA=";
+      excludes = [ "Source/WebKit/ChangeLog" ];
+    })
+
+    # https://bugs.webkit.org/show_bug.cgi?id=225856
+    (fetchpatch {
+      url = "https://bug-225856-attachments.webkit.org/attachment.cgi?id=428797";
+      sha256 = "sha256-ffo5p2EyyjXe3DxdrvAcDKqxwnoqHtYBtWod+1fOjMU=";
+      excludes = [ "Source/WebCore/ChangeLog" ];
+    })
+
+    # https://bugs.webkit.org/show_bug.cgi?id=225850
+    ./428774.patch # https://bug-225850-attachments.webkit.org/attachment.cgi?id=428774
```

Comment:
Why not fetch the patch?

Reply:
Because it won't apply, and an additional change to IPCSemaphore.cpp is needed for the current version.

Reply:
In this case it would be nice to include a note in the file's header about where the patch came from and what modifications you made.

Reply:
I'm the author of the both patches. I can split the change to IPCSemaphore.cpp into a separate patch (although, I think, logically it belongs to the same patch) and comment on it, if you think that would be helpful.